app/views/submissions/show.erb

```diff
@@ -41,6 +41,13 @@
       submitted by <%= gravatar_tag submission.user.avatar_url, size: 20 %>
       <%= profile_link(submission.user) %>
       <%= ago(submission.created_at) %>
+      <br/>
+      see other implementations of
+      <a href="<%= "/tracks/#{submission.problem.track_id}/exercises/#{submission.problem.slug}" %>">
```

Comment:
a couple nits: 
`track_id` is accessible directly from `submission`, and likewise so is `slug`